strawberry/annotation.py

```diff
@@ -179,13 +179,13 @@ def create_concrete_type(self, evaled_type: type) -> type:
             return evaled_type.__strawberry_definition__.resolve_generic(evaled_type)
         raise ValueError(f"Not supported {evaled_type}")
 
-    def create_enum(self, evaled_type: Any) -> EnumDefinition:
+    def create_enum(self, evaled_type: EnumMeta) -> EnumDefinition:
         try:
             return evaled_type._enum_definition
         except AttributeError:
             raise NotAStrawberryEnumError(evaled_type)
 
-    def create_list(self, evaled_type: Any) -> StrawberryList:
+    def create_list(self, evaled_type: TypeVar) -> StrawberryList:
```

Comment:
are you sure these are typevars?

Reply:
Ops, I double checked and most seem to be `type`: https://github.com/strawberry-graphql/strawberry/pull/3526/commits/a31f23d933f7e691691f406539245c6b158cc622